cyder/cydhcp/network/models.py

```diff
@@ -288,6 +287,16 @@ def descendants(self):
                 'Network.descendants does not currently support IPv6')
 
     @property
+    def children(self):
+        self.update_network()
+        descendants = set(self.descendants)
+        children = []
+        for d in descendants:
+            if d.parent not in descendants:
```

Comment:
I'm not sure what I was thinking. I don't think it's safe to use `d.parent is self`, but `d.parent == self` should be fine.